hack/ref_links.lua

```diff
@@ -0,0 +1,39 @@
+local anchors = {}
+
+function Header(el)
+  if el.level == 3 and el.identifier then
+    local ref_name = el.identifier:match("[^.]+$")
+    el.identifier = ref_name
+    return {
+      el
+    }
+  end
+end
+
+function Link(el)
+  local href = el.target
+  if href:match("^https://kubernetes%.io") then
+    return el  -- Do nothing, return the original link
+  end
+  if href:match("^https://godoc") then
+    return el  -- Do nothing, return the original link
+  end
+  -- Check if the link is "#mellanox.com%2fv1alpha1"
+  if href:match("mellanox%.com%%2fv1alpha1") then
```

Comment:
if we remove this, we remove the links for the list of packages. Can we keep it?

Reply:
Need to convert it to normal string and see if it works.
I filtered it out because it was not working.
And for now, the list is just one package...